CHANGELOG.next.asciidoc

```diff
@@ -98,6 +98,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Filebeat*
 
 - Fix using log_group_name_prefix in aws-cloudwatch input. {pull}29695[29695]
+- Fix journald event format
```

Comment:
Has this bug ever been included in a release? If we caught it before it was released then omit the changelog entry.

Reply:
No it haven't, I'll remove it